### PR TITLE
[FIX] Missing i18n translation key for "Unread"

### LINKED ISSUE #10385: Sidebar: Unread Label is "hardcoded" leading to wrong translations
### Description:

If you sort your channels like "unread on top" the label of the menu group seems to be hardcoded, which leads to wrong translations in other languages. (i.e. Label is also "Unread" if I select german as my language)

### Server Setup Information:

* Version of Rocket.Chat Server: 0.63.1
* Operating System: Ubuntu 16.04 LTS
* Deployment Method(snap/docker/tar/etc):  docker
* Number of Running Instances: 3

### Steps to Reproduce:

1. Enable "unread channels on top"
2. Select another language as english
3. The label is wrong

Screenshot:

image

### Expected behavior:

The label for the group of unread channels should be a variable so it translates correctly to the language which the users choose.

### Actual behavior:

The label is english :)

### Code:

I think this could be the section there it is hardcoded:

https://github.com/RocketChat/Rocket.Chat/blob/41bffeadbc6dd6eaa1c3b486442b1232670bb477/packages/rocketchat-lib/lib/roomTypes/unread.js#L8

### PR DESCRIPTION
**Pull request by Hudell**
Closes #10385 

Added missing "Unread" key to the i18n files.

### CODE REVIEW

File: packages/rocketchat-i18n/i18n/pt-BR.i18n.json [hunk 1]
```diff
@@ -2011,6 +2011,7 @@
   "Unmute_user": "Permitir que o usuário fale",
   "Unnamed": "Sem nome",
   "Unpin_Message": "Desafixar Mensagem ",
+  "Unread": "Mensagens Não Lidas",
```
Comment (sampaiodiego) [inline packages/rocketchat-i18n/i18n/pt-BR.i18n.json]: the correct translation is just `Não Lidas` since it is used to show unread rooms, not messages.

Comment (Hudell) [reply packages/rocketchat-i18n/i18n/pt-BR.i18n.json]: Fixed it.

**Comment by kaiiiiiiiii** [comment]
Can you change the german translation as well, please? As @sampaiodiego said, it’s about unread channels/rooms, not messages. The right german translation would be “Ungelesene Kanäle” or much more simple “Ungelesen” (which is the equivalent of unread).

Thanks

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by cadetmsv** [comment]
I would also like to Missing i18n translation key for "Unread_on_top" don't translate

**Pull request merged**